instruments/instruments/qubitekk/cc1.py

```diff
@@ -52,8 +65,11 @@ def __init__(self, filelike):
         self.terminator = "\n"
         self.end_terminator = "\n"
         self.channel_count = 3
+        self.channel1 = self.Channel(self, 02)
```

Comment:
You already have `CC1.channel` defined as a property using `ProxyList`, just remove these.